.github/workflows/label-zero-impact.js

```diff
@@ -0,0 +1,68 @@
+const { getLocalConfigs } = require('./helpers.js');
+const zeroImpactDirs = ['.github', '.kodiak', '.vscode', '.test', 'libs/mep'];
```

Comment:
@vgoodric can you confirm that `libs/mep` is special and never touches prod code or basically can be included in stage/prod at any point in time during IST/EU/US times?